db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+if Doorkeeper::Application.count.zero?
+  Doorkeeper::Application.create!(name: 'Web Client', redirect_uri: '', scopes: '')
+  Doorkeeper::Application.create!(name: 'iOS Client', redirect_uri: '', scopes: '')
+  Doorkeeper::Application.create!(name: 'Android Client', redirect_uri: "", scopes: '')
+end
```

Comment:
Optional suggestion: Should we also use `find_or_create_by` for the `DoorKeeper::Applications` seeds?